backend/Squirrel.Core/Squirrel.Core.WebAPI/appsettings.json

```diff
@@ -2,6 +2,10 @@
   "ConnectionStrings": {
     "SquirrelCoreDBConnection": "Server=localhost;Database=SquirrelCoreDB;Trusted_Connection=True;"
   },
+  "MongoDatabase": {
+    "ConnectionString": "",
+    "DatabaseName": ""
+  },
```

Comment:
I think, it would probably be better to move this to the `ConnectionStrings` block and get rid of the `MongoDataBaseConnectionSettings` model.

Reply:
I agree with the placement, but what is the benefit of removing MongoDatabaseConnectionSettings?

Reply:
It's ok to leave it here